packages/riipen-ui/src/components/TableCell.test.jsx

```diff
@@ -0,0 +1,246 @@
+    });
+  });
+
+  describe("classes prop", () => {
+    it("sets custom class", () => {
+      const classes = ["newClass"];
+
+      const wrapper = mount(
+        <Table>
+          <TableBody>
+            <TableRow>
+              <TableCell classes={classes} />
+            </TableRow>
+          </TableBody>
+        </Table>
+      );
+
+      expect(
+        wrapper
+          .find("TableCell")
+          .childAt(0)
+          .hasClass(classes[0])
+      ).toEqual(true);
+    });
+
+    it("appends higher order values to classes prop with withClasses decorator", () => {
+      const classes = ["riipen", "riipen-tablecell"];
+
+      const wrapper = mount(
+        <Table>
+          <TableBody>
+            <TableRow>
+              <TableCell />
+            </TableRow>
+          </TableBody>
+        </Table>
+      );
+
+      expect(
+        wrapper
+          .find("TableCell")
+          .props()
+          .classes.sort()
+      ).toEqual(classes.sort());
+    });
+  });
+
+  describe("colSpan prop", () => {
+    it("sets custom colSpan", () => {
+      const colSpan = 2;
+
+      const wrapper = mount(
+        <Table>
+          <TableBody>
+            <TableRow>
+              <TableCell colSpan={colSpan} />
+            </TableRow>
+          </TableBody>
+        </Table>
+      );
+
+      expect(
+        wrapper
+          .find("TableCell")
+          .find("td")
+          .props().colSpan
+      ).toEqual(colSpan);
+    });
+  });
+
+  describe("padding prop", () => {
+    it("sets custom padding", () => {
+      const padding = 2;
+
+      const wrapper = mount(
+        <Table>
+          <TableBody>
+            <TableRow>
+              <TableCell padding={padding} />
+            </TableRow>
+          </TableBody>
+        </Table>
+      );
+
+      expect(
+        wrapper
+          .find("TableCell")
+          .childAt(0)
+          .props().style.padding
+      ).toContain(`${defaultTheme.spacing(padding)}px`);
+    });
+  });
+
+  describe("rowSpan prop", () => {
+    it("sets custom rowSpan", () => {
+      const rowSpan = 2;
+
+      const wrapper = mount(
+        <Table>
+          <TableBody>
+            <TableRow>
+              <TableCell rowSpan={rowSpan} />
+            </TableRow>
+          </TableBody>
+        </Table>
+      );
+
+      expect(
+        wrapper
+          .find("TableCell")
+          .find("td")
+          .props().rowSpan
+      ).toEqual(rowSpan);
+    });
+  });
+
+  describe("textAlign prop", () => {
+    it("sets custom valid textAlign", () => {
+      const textAlign = "center";
+
+      const wrapper = mount(
+        <Table>
+          <TableBody>
+            <TableRow>
+              <TableCell textAlign={textAlign} />
+            </TableRow>
+          </TableBody>
+        </Table>
+      );
+
+      expect(
+        wrapper
+          .find("TableCell")
+          .childAt(0)
+          .props().style.textAlign
+      ).toContain(textAlign);
+    });
```

Comment:
It might be good to change the assertion to `.toEqual(textAlign)`? `textAlign` shouldn't be a substring of the `style.textAlign`.

Reply:
Thanks, adjusted! Typo :(